### ISSUE TITLE
Implement Google OIDC Session Management

### ISSUE THREAD

**Issue by jricher** [mitreid-connect/OpenID-Connect-Java-Spring-Server#218]

**Comment by almalki** [comment]
Hi Justin,

I am thinking of implementing the session management for Mitreid OIDC server. After a quick research I found out that the OpenID Connect Session Management 1.0 spec is still a draft, 
https://openid.net/specs/openid-connect-session-1_0.html

Is this the correct spec to implement for this feature? and why is Google is mentioned here?

Thanks..

**Comment by jricher** [comment]
Yes, that is the correct specification, and it is still a draft (non-final) standard. Google has implemented a version of that spec, but I think even their implementation differs.

**Comment by almalki** [comment]
Hi @jricher 

I started implementing the draft spec, but I was stuck here:

https://github.com/spring-projects/spring-security-oauth/issues/549

I appreciate your input ion this issue.
